### PR TITLE
[Commands]: Add new commands

### PR DESCRIPTION
**Pull request by fallaciousreasoning**
Resolves https://github.com/brave/brave-browser/issues/31064
Resolves https://github.com/brave/brave-browser/issues/31298

Adds new commands for:
1. Toggling vertical tabs expansion
2. Moving all tabs on an origin into a group
3. Moving a group to a new window
4. Closing duplicate tabs

## Submitter Checklist:

- [ ] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [ ] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [ ] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [ ] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [ ] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [ ] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally:
  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

### CODE REVIEW

File: browser/ui/browser_commands.cc [hunk 1]
```diff
@@ -254,6 +260,12 @@ void ToggleVerticalTabStripFloatingMode(Browser* browser) {
       !prefs->GetBoolean(brave_tabs::kVerticalTabsFloatingEnabled));
 }
 
+void ToggleVerticalTabStripExpanded(Browser* browser) {
+  auto* prefs = browser->profile()->GetOriginalProfile()->GetPrefs();
```
Comment (sangwoo108) [inline browser/ui/browser_commands.cc:264]: As kVerticalTabsCollapsed is added to `brave_pref_service_incognito_allowlist.cc`, I hope we can remove `GetOriginallProfile()` here.

File: browser/ui/browser_commands.cc [hunk 2]
```diff
@@ -342,4 +354,53 @@ void ShowPlaylistBubble(Browser* browser) {
...
+  }
+
+  tsm->delegate()->MoveGroupToNewWindow(current_group_id.value());
+}
+
+void CloseDuplicateTabs(Browser* browser) {
+  auto* tsm = browser->tab_strip_model();
+  auto url = tsm->GetActiveWebContents()->GetVisibleURL();
+
+  for (int i = tsm->GetTabCount() - 1; i >= 0; --i) {
```
Comment (sangwoo108) [inline browser/ui/browser_commands.cc:393]: Out of curiosity, is there any special reason to iterate in the reverse order?

**Review by sangwoo108 — APPROVED** [review]
LGTM 👍

File: browser/ui/browser_commands.cc (hunk 2 above)
Comment (sangwoo108) [reply browser/ui/browser_commands.cc:393]: Aha! is it because the index could be changed when closing tabs?

**Review by mkarolin — APPROVED** [review]
strings: I guess it's ok to keep doing this with automatic string generation for now, but these strings tend to end up not being well formed grammatically an cause a lot of questions from localizers. 
++

File: browser/ui/browser_commands.cc (hunk 1 above)
Comment (fallaciousreasoning) [reply browser/ui/browser_commands.cc:264]: Good call!

File: browser/ui/browser_commands.cc (hunk 2 above)
Comment (fallaciousreasoning) [reply browser/ui/browser_commands.cc:393]: Yeah, exactly!

**Pull request merged**